ab2d-events-client/src/test/java/gov/cms/ab2d/eventclient/SpringBootApp.java

```diff
@@ -5,14 +5,7 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
 
-@SpringBootApplication(
-        scanBasePackages = {"gov.cms.ab2d.eventclient"},
-        exclude = {
-                org.springframework.cloud.aws.autoconfigure.context.ContextInstanceDataAutoConfiguration.class,
-                org.springframework.cloud.aws.autoconfigure.context.ContextStackAutoConfiguration.class,
-                org.springframework.cloud.aws.autoconfigure.context.ContextRegionProviderAutoConfiguration.class
-        }
-)
+@SpringBootApplication(scanBasePackages = {"gov.cms.ab2d.eventclient"})
```

Comment:
Need to check and see what the ramifications of not having these excluded are. There may be some workarounds, but removing this was required for a successful build locally.